vending-app/src/main/res/values/strings.xml

```diff
@@ -60,4 +60,11 @@
     <string name="billing_permission_desc">Allows the user to purchase items through Google Play from within this app.</string>
     <string name="billing_permission_label">Google Play billing service</string>
 
+    <string name="channel_install_permission_allowed">Allow</string>
+    <string name="channel_install_needs_permissions_reminder_title">Necessary authorization reminder</string>
+    <string name="channel_install_allow_to_install_third_app">Allow \"microG Companion\" to install apps downloaded from \"%1$s\"</string>
+    <string name="channel_install_refuse_permission_remind">If you decline, the application installation will fail.</string>
+    <string name="channel_install_refuse_permission_remind_sub">You can modify the authorization status permissions at any time in \"Settings > Apps And Services > App Management > microG Service > microG Settings Page\".</string>
```

Comment:
The user interface to get the microG Settings is different between various Android versions. I think easiest would be to just write "microG Settings" here and have the user figure out themselves.